client/src/components/frontend/Resource/TagList.js

```diff
@@ -34,14 +34,14 @@ export default class ResourceTagList extends Component {
     event.stopPropagation();
   };
 
-  createTagLink(tag, projectId, index) {
-    if (!tag || !projectId) return null;
+  createTagLink(tag, projectSlug, index) {
+    if (!tag || !projectSlug) return null;
     return (
       <li key={`${tag}-${index}`}>
         <Link
           onClick={this.stopPropagation}
           className={this.props.disabledLinks ? "disabled" : null}
-          to={`/project/${projectId}/resources?tag=${tag.toLowerCase()}`}
+          to={`/project/${projectSlug}/resources?tag=${tag.toLowerCase()}`}
```

Comment:
This is a no-no — please use a link helper here.

Reply:
Good call, missed that